.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
   - 3.3
   - 3.4
   - 3.5
-  - 3.6
```

Comment:
Have you tried something like
```
# tox.ini
[tox:travis]
nightly = py36
```
?  I'm not saying it would work, just curious if I'm interpreting https://github.com/ryanhiebert/tox-travis#advanced-configuration correctly.